server/prisma/datamodel.prisma

```diff
@@ -3,9 +3,31 @@ type User {
   email: String! @unique
   password: String!
   name: String!
+  campaigns: [Campaign!]!
 }
 
-type Post {
+type Event{
+   id: ID! @id
+   title: String! @unique
+   createdAt: DateTime! @createdAt
+   location: String!
+   image: String!
+   description: String!
+   comments: [Comment!]!
```

Comment:
Do you need a field for Campaign here?

Reply:
Yes, the user has MyCampaign screen which is a list of their campaigns.

Reply:
If you need to see what campaign this event belongs to then please add a field for a campaign here. Thank you.